e2e/datpicker-e2e.spec.ts

```diff
@@ -183,11 +183,11 @@ describe('dpDayPicker dayPicker', () => {
 
   it('should check the first day of the week', () => {
     page.dayPickerInput.click();
-    expect(page.weekDayNames.getText()).toEqual(['Sun Mon Tue Wed Thu Fri Sat']);
+    expect(page.weekDayNames.getText()).toEqual(['SunMonTueWedThuFriSat']);
```

Comment:
why did it effect this?

Reply:
it seems there was whitespace inside the tags before,  like:
```html
<span>
  November
</span>
```

which now becomes:
```html
<span>November</span>
```